server.js

```diff
@@ -11,7 +11,7 @@ var urlencodedParser = bodyParser.json({ extended: false });
 var TokenClient = require('@token-io/tpp').TokenClient; // main Token SDK entry object
 
 // Connect to Token's development sandbox, if you change this, you also need to change window.Token({env}) in client.js
-var Token = new TokenClient({ env: 'sandbox', developerKey: '4qY7lqQw8NOl9gng0ZHgT4xdiDqxqoGVutuZwrUYQsI', keyDir: './keys' });
+var Token = new TokenClient({ env: 'dev', developerKey: '4qY7lqQw8NOl9gng0ZHgT4xdiDqxqoGVutuZwrUYQsI', keyDir: './keys' });
```

Comment:
Why do we change this to dev? Would you need to change the developerKey (to make sure this works for dev)?

Reply:
No, I don't think there is a need to change the key. I think it's same for all env unless we pass some other

Reply:
Yeah but why are we changing this from sandbox to dev?

Reply:
popup + one step breaks Uncaught (in promise) EXCEPTION_INVALID_BANK: Could not find bank with ID ngp-cbi-05034

This bank is in sandbox only: https://github.com/tokenio/configurations/search?q=ngp-cbi-05034&unscoped_q=ngp-cbi-05034
Pls test locally